packages/web-twig/src/Resources/components/Breadcrumbs/index.twig

```diff
@@ -0,0 +1,37 @@
+{# API #}
+{%- set _class = props.class | default(null) -%}
+{%- set _elementType = props.elementType | default('nav') -%}
+{%- set _goBackItemTranslation = props.goBackItemTranslation | default('Back') -%}
+{%- set _items = props.items | default([]) -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'Breadcrumbs' -%}
+
+{# Miscellaneous #}
+{%- set _classNames = [ _rootClassName, _class ] -%}
+
+<{{ _elementType }} {{ mainProps(props) }} {{ classProp(_classNames) }} aria-label="Breadcrumb">
+{% apply spaceless %}
+    {%- if items|length > 0 -%}
+        <ol>
+            {% for item in items %}
+                {% if loop.index is same as(items|length - 1) %}
+                    <li class="d-tablet-none">
+                        <a href="{{ item.url }}">{{ _goBackItemTranslation }}</a>
+                    </li>
+                {% endif %}
+                <li class="d-none d-tablet-flex">
+                    <a
+                        href="{{ item.url }}"
+                        {% if loop.last %}
+                            aria-current="page"
+                        {% endif %}
+                    >{{ item.title }}</a>
+                </li>
+            {% endfor %}
+        </ol>
+    {%- else -%}
+        {%- block content -%}{%- endblock -%}
```

Comment:
Please, be consistent with `web-react` `Breadcrumbs` component. Either allow this also in React component or remove it from here.

Reply:
Allowed in both.